image_client/PIC_undo_batch.py

```diff
@@ -106,26 +137,24 @@ def casbotany_csv_purger(self, database: str, table: str, MD5: str):
 
         table_list = ['collectionobjectattachment', 'attachment',
                       'determination', 'collectionobject', 'collector',
-                      'collectingevent', 'locality', 'agent', ]
+                      'collectingevent', 'locality', 'agent']
 
         for table in table_list:
-            self.sql_time_purger(database='casbotany', table=table,
-                                 timestamp1=time_stamp_list[0],
-                                 timestamp2=time_stamp_list[1])
+            self.batch_undo_timestamps(database='casbotany', table=table,
+                                       timestamp1=time_stamp_list[0],
+                                       timestamp2=time_stamp_list[1])
 
         table = "taxon"
 
-        self.sql_tree_purger(database='casbotany', table=table,
-                             timestamp1=time_stamp_list[0],
-                             timestamp2=time_stamp_list[1])
-
-    def run_all(self):
-        self.casbotany_csv_purger(database="casbotany", table="picturae_batch", MD5=self.purge_code)
+        self.batch_tree_pruner(database='casbotany', table=table,
+                               timestamp1=time_stamp_list[0],
+                               timestamp2=time_stamp_list[1])
 
+        # clearing picbatch records
+        for table in ['picturaetaxa_added', 'taxa_unmatch', 'picturae_batch']:
```

Comment:
uses new batch_log_clear function, uses hardcoded tab names to match what is in the DDL.